### PR TITLE
Add GKE Alias IP documentation

### PR DESCRIPTION
**Pull request by msau42**
As a recommendation to avoid IP conflicts

### CODE REVIEW

**Comment by msau42** [comment]
assign @bowei

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-    * IP reservation for this driver is a future enhancement.
-    * GKE Pod and Service CIDRs can be reserved during cluster creation using the
-      `--cluster-ipv4-cidr` and `--services-ipv4-cidr` flags.
-    * GKE TPU CIDRs can be reserved during cluster creation using the
-      `--tpu-ipv4-cidr` flag.
+  reserve IP ranges, such as GKE non-IP alias clusters or GKE TPUs. To avoid
+  IP conflicts, it is recommended to either:
+    * Use a GKE cluster with [Alias IPs](https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips)
+        * This will prevent IP conflicts with GKE Pod and Service IPs, but not TPUs.
+    * Explicitly allocate IP ranges to each GCP service, and this plugin.
```
Comment (davidz627) [inline KNOWN_ISSUES.md]: link to documentation for this?

Comment (msau42) [reply KNOWN_ISSUES.md]: added

**Pull request merged**